contents/blog/ceo-diary-1.md

```diff
@@ -0,0 +1,70 @@
+---
+date: 2021-08-19
+title: Winning from the back - CEO Diary 1
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: james-hawkins
+---
+
+We are open source is [literally our top value](../handbook/company/values)... and what better way to be transparent than to share a diary!
+
+# Late mover advantage
+
+Product analytics is a busy space. It reminds me of the _front_ entrance to Liverpool Street station in London:
+
+![Liverpool Street Front Entrance - lots of people](../images/blog/CEO-diary-1/liverpool-st-station.jpg)
+
+PostHog Fact: Tim (my cofounder) incorrectly believes this is the side entrance - it's one of our greatest disagreements.
```

Comment:
Obvious place to for a implicit pineapple joke ("second greatest disagreement", for example)

Reply:
I literally had this in there to start with but felt it was starting to detract (given I'm already out on a meander by talking about trains)

Reply:
Yeah, thought it was too obvious :D. How about a separate page in the handbook regarding what Tim and James disagree on? Or a `[citation needed]` wikipedia reference? Fine to leave it at trains ofc. :)